### DISCUSSION TITLE
2009/06/10/express-vi

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#268] [Announcements]
# 2009/06/10/express-vi

更新内容在： https://lv.qizhen.xyz

https://blog.qizhen.xyz/2009/06/10/express-vi

**Comment by ruanqizhen** [comment]
之前博客下的留言：

川
2009年07月7日 12:29 编辑
阮老师，谢谢你的回复。祝你工作愉快，写更多的好文章。

回复
Qizhen
2009年07月6日 15:30 编辑
我没用过Matlab。建议你试着使用C语言调用一下看看，先确定是否是你生成的DLL有问题。

回复
川
2009年07月6日 12:45 编辑
阮老师，你好！经常访问你的空间学习，真是受益不浅。阮老师，在LabVIEW中调用Matlab生成的dll的方法你能否费心讲一下？在NI或者百度上我都找过方法了，试验过都不成功。谢谢！

回复

fengtop
2009年07月15日 00:23 编辑
感谢分享!

回复
